pep257.py

```diff
@@ -254,6 +268,47 @@ def parse_docstring(self):
             return docstring
         return None
 
+    def parse_decorators(self):
+        """Called after first @ is found.
+
+        Build list of current decorators and return last parsed token,
+        which is def or class start token.
+        """
+        name = []
+        arguments = []
+        at_arguments = False
+
+        while self.current is not None:
+            if (self.current.kind == tk.NAME and
+                    self.current.value in ['def', 'class']):
+                break
+            elif self.current.kind == tk.OP and self.current.value == '@':
+                # New decorator found.
```

Comment:
This comment confused me, because I didn't understand how a new decorator is appended if it's just been found out. State that you're saving the _old_ one and resetting the parameters.